aws/resource_aws_cloudfront_distribution_test.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/aws/aws-sdk-go/service/cloudfront"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```

Comment:
Similarly here, this testing did not need extra providers or to save the instances.